src/OrchardCore.Themes/TheAdmin/Views/Layout.cshtml

```diff
@@ -8,14 +8,13 @@
     <link type="image/x-icon" rel="shortcut icon" href="~/TheAdmin/favicon.ico" />
 
     <!-- Bootstrap CSS -->
-    <style asp-src="~/TheAdmin/Styles/TheAdmin.min.css" debug-src="~/TheAdmin/Styles/TheAdmin.css" ></style>
-
+    <style asp-src="~/TheAdmin/Styles/TheAdmin.min.css" debug-src="~/TheAdmin/Styles/TheAdmin.css"></style>
+    <style asp-src="~/OrchardCore.Resources/Styles/jquery-ui.min.css"></style>
```

Comment:
Actually sorry @hishamco this still won't respect the resource manager settings.
You got the script but not the style!

```
<style asp-name="admin" asp-src="~/TheAdmin/Styles/TheAdmin.min.css" debug-src="~/TheAdmin/Styles/TheAdmin.css" depends-on="jQuery-ui"></style>
```

This is best (because I see in some of the other views, like src/OrchardCore.Modules/OrchardCore.AdminMenu/Views/Items/LinkAdminNode.Fields.TreeEdit.cshtml )
that the styles have a depenency on admin, but we don't actually name it anywhere)